.NET/Microsoft.Recognizers.Text.Sequence/English/Extractors/PhoneNumberExtractor.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Text.RegularExpressions;
+
+using Microsoft.Recognizers.Definitions;
+
+namespace Microsoft.Recognizers.Text.Sequence.English
+{
+    class PhoneNumberExtractor : BaseSequenceExtractor
+    {
+        internal override ImmutableDictionary<Regex, string> Regexes { get; }
+
+        protected sealed override string ExtractType { get; } = Constants.MODEL_PHONE_NUMBER;
+
+        public PhoneNumberExtractor()
+        {
+            var regexes = new Dictionary<Regex, string>
```

Comment:
Shouldn't these be in a BasePhoneNumberExtractor?

Reply:
Yes, I agree. But does BasePhoneNumberExtractor inherit from BaseSequenceExtractor? Because of the same extract logic they share.